filebeat/harvester/log.go

```diff
@@ -74,7 +74,7 @@ func (h *Harvester) Harvest() {
 
 				h.SetOffset(0)
 				seeker.Seek(h.getOffset(), os.SEEK_SET)
-				continue
+				return
```

Comment:
The complete `if err == errFileTruncate` can be removed. Where/how does prospector detect the file being truncated? Still `h.SetOffset(0)` required?